package.json

```diff
@@ -9,7 +9,7 @@
     "build": "tsc",
     "watch": "tsc --w",
     "prestart": "npm run build && func extensions install --javascript",
-    "start:host": "func start --javascript",
+    "start:host": "dotenv -e .env func start --javascript",
```

Comment:
there's no need to feed .env since it's read automatically on startup when running func

Reply:
Maybe it was a misconfiguration of mine :) but when I ran `yarn start` without dotenv, it seems it doesn't work properly. In order to make it work I' have to export all the variables included in `.env` file with `export $(grep -v '^#' .env | xargs )`.

Reply:
It's because you are not initializing dotenv.

`require('dotenv').config()`

Also take in mind that dotenv is a devDependency so you lose it when install with --production.

Reply:
I'll open another PR :)

Reply:
I just checked that you are using dotenv-cli and not dovenv :)